README.md

```diff
@@ -1,70 +1,114 @@
-# Getting Started with Create React App
+<p align="center">
+  <a href="https://www.microverse.org/">
+    <img alt="Microverse" src="https://img.shields.io/badge/-Microverse-blueviolet?style=flat-square">
+  </a>
+  <a href="https://github.com/Emmanuelaaron/spaceships">
+    <img alt="GitHub last commit (branch)" src="https://img.shields.io/github/last-commit/Emmanuelaaron/spaceships/dev?color=blue&style=flat-square">
+  </a>
+  <a href="https://github.com/Emmanuelaaron/spaceships">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/Emmanuelaaron/spaceships?color=cyan&label=%E2%98%85%20stars%20&style=flat-square">
+  </a>
+</p>
```

Comment:
OPTIONAL

- [ ] Add the live link to the deployed version of your app to this file.